desktop/src/main/java/bisq/desktop/main/funds/locked/LockedView.java

```diff
@@ -219,6 +237,61 @@ protected void deactivate() {
     // Private
     ///////////////////////////////////////////////////////////////////////////////////////////
 
+    private void applyFilteredListPredicate(String filterString) {
+        filteredList.setPredicate(item -> {
+            if (filterString.isEmpty())
+                return true;
+
+            if (item.getDetails().contains(filterString)) {
+                return true;
+            }
+
+            if (item.getDateString().contains(filterString)) {
+                return true;
+            }
+
+            if (item.getAddressString().contains(filterString)) {
+                return true;
+            }
+
+            if (item.getBalanceString().contains(filterString)) {
+                return true;
+            }
+
+            Trade trade = item.getTrade();
+            if (trade != null) {
+                if (trade.getTakerFeeTxId() != null && trade.getTakerFeeTxId().contains(filterString)) {
+                    return true;
+                }
+                if (trade.getDepositTxId() != null && trade.getDepositTxId().contains(filterString)) {
+                    return true;
+                }
+                if (trade.getPayoutTxId() != null && trade.getPayoutTxId().contains(filterString)) {
+                    return true;
+                }
+
+                Contract contract = trade.getContract();
+
+                boolean isBuyerOnion = false;
+                boolean isSellerOnion = false;
+                boolean matchesBuyersPaymentAccountData = false;
+                boolean matchesSellersPaymentAccountData = false;
+                if (contract != null) {
+                    isBuyerOnion = contract.getBuyerNodeAddress().getFullAddress().contains(filterString);
```

Comment:
Those predicates appear several times. Would be good to make it reusable code

Reply:
[Fixed](https://github.com/bisq-network/bisq/pull/5963/commits/2f6c249e3197f8f2a007b717daf6132b3bf7fed9)